app.R

```diff
@@ -371,6 +373,11 @@ ui <- fluidPage(
                         actionButton("resetBatchSliders", "Reset Values!"),
                         br(),
                         
+                        # Input: Threat reminder
+                        
+                        checkboxInput("threatvalue", label = "No observed, estimated, projected, inferred, or suspected declines likely 
+                                       to trigger criteria A, B, C, D or E." , value = FALSE),
+                        
```

Comment:
Add something here to nudge user to tick the box so they can unlock the download button e.g. Tick box to confirm - maybe we can sort this with review of batch UI?

Reply:
I think this would be something to do for a wider review of the batch tab UI.